src/style.css

```diff
@@ -1 +1,71 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  background-position: center;
+}
+
+.stars--0 .stars__star {
+  background-image: url(/src/images/star.svg);
+}
+
+.stars--5 .stars__star {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--1 .stars__star:first-child {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--1 .stars__star:nth-child(2),
+.stars--1 .stars__star:nth-child(3),
+.stars--1 .stars__star:nth-child(4),
+.stars--1 .stars__star:nth-child(5) {
+  background-image: url(/src/images/star.svg);
+}
+
+
+
+.stars--2 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(2) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(3),
+.stars--2 .stars__star:nth-child(4),
+.stars--2 .stars__star:nth-child(5) {
+  background-image: url(/src/images/star.svg);
```

Comment:
it's too complicated, you can just use something like

`.stars--2 .stars__star:nth-child(-n + 2),`

Reply:
fixed